quinn/examples/connection.rs

```diff
@@ -1,18 +1,3 @@
-//! This example intends to use the smallest amount of code to make a simple QUIC connection.
```

Comment:
I'm not a fan of outright deleting this. Can we keep 2-3 lines of description and point to the README?

Reply:
I can do that, will introduce a bit of duplication which is a tradeoff to keep in mind.

Reply:
That's fair. It's just, people might "get" to this file in different ways, and it would be bad if someone found it without having seen the README and would then be in worse position than they would be now.